src/components/PlayClassCard/PlayClassCard.js

```diff
@@ -35,9 +24,9 @@ export default function PlayClassCard(props) {
   }
 
   return (
-    <section className="PlayClass" {...rest}>
+    <section className="PlayClass">
       <IoHeart
-        className={isFavourite ? "FavIcon--inactive" : "FavIcon--active"}
+        className={isFavourite ? "FavIcon--active" : "FavIcon--inactive"}
```

Comment:
I like what you did. But do you really need an inactive modifier or would it be easier to just have inactive as the default class and active as a modifier?

Reply:
It is inactive per default. This is toggling back and forth.